src/main/java/com/numble/karrotmarket/product/controller/ProductController.java

```diff
@@ -24,7 +25,7 @@ public List<Product> getProducts() {
     }
 
     @PostMapping
-    public Product createProduct(@RequestBody final ProductCreationRequest request) {
+    public Product createProduct(@RequestBody @Validated final ProductCreationRequest request) {
```

Comment:
`@Validated`를 클래스에 정의를 하고 `@Valid`를 파라미터에 정의를 하는 것은 어떨까요?

```suggestion
    @PostMapping
    @Validated
    public Product createProduct(@RequestBody @Valid final ProductCreationRequest request) {
```

Reply:
AOP보다는 ArgumentNameResolver방식이 훨씬 더 좋아 보입니다. 따라서 @Validated -> @Valid 변경 필요해 보이네요. 감사합니다.

다만 Contoller 클래스에 있기 때문에 ArgumentNameResolver 동작시 @Valid 어노테이션을 캐치해서 Validation 하고 있기 때문에 @Validated 선언은 필요 없을 것 같습니다.